CSS/about.css

```diff
@@ -0,0 +1,1030 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+}
+
+body {
+  display: grid;
+  grid-template-columns: auto;
+  background-color: #fff;
+  overflow-x: hidden;
+}
+
+/* Hero Section Starts */
+
+#hero {
+  width: 100vw;
+}
+
+.header-wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.pre-header {
+  display: none;
+}
+
+#main-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100vw;
+  height: 73px;
+  background: #fff;
+  box-shadow: 4px 4px 4px 0 rgba(0, 0, 0, 0.25);
+  position: relative;
+}
+
+.hamburger-container {
+  width: 31px;
+  height: 27px;
+  margin-left: 23px;
+  transition: 0.1s;
+}
+
+#hamburger-link {
+  width: 100%;
+  height: 100%;
+}
+
+.hamburger {
+  width: 100%;
+  height: 100%;
+}
+
+.hamburger-container:hover {
+  transform: scale(1.05);
+}
+
+.logo-container {
+  width: 130px;
+  height: 51px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  -ms-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+}
+
+.logo-link {
+  width: 100%;
+  height: 100%;
+}
+
+.logo {
+  width: 100%;
+  height: 100%;
+}
+
+#navbar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  right: 100%;
+  width: 100vw;
+  height: 100vh;
+  background-color: #f8f8f8;
+  z-index: 3;
+  padding: 30px;
+  transition: 0.3s;
+}
+
```

Comment:
- [ ] Wow, your design sense is truly remarkable! 👏👏 However, I noticed qn issue  in the mobile menu for the "About" page. There seems to be some text overlapping, which could potentially affect the user experience and readability on smaller screens.. You can try fixing this by increasing the value of the `z-index`

![image](https://github.com/Al-Amin-Khan-Shakil/FinanceHorizons-Business-Conference/assets/106119105/07f35161-c8e9-47c2-8772-01824c03df45)